docs/install.md

```diff
@@ -74,3 +87,39 @@ The different supported modes can be set by using the `hook-mode` option:
 - `podinformer`: Use a Kubernetes controller to get information about new pods. This option is racy and the first events produced by a container could be lost. This mode is selected when `auto` is used and the above modes are not available.
 - `ldpreload`: Adds an entry in `/etc/ld.so.preload` to call a custom shared library that looks for `runc` calls and dynamically adds the needed OCI hooks to the cointainer `config.json` specification. This feature only works when runc is used. Since this feature is highly experimental, it'll not be considered when `auto` is used.
 - `nri`: Use the [Node Resource Interface](https://github.com/containerd/nri). It requires containerd v1.5 and it's not considered when `auto` is used.
+
+### Specific Information for Different Platforms
+
+This section explains the additional steps that are required to run Inspektor
+Gadget in some platforms.
+
+#### Minikube
+
+You should create the minikube cluster in different ways according to the gadget
+you want to use. If you want to use traceloop or the network policy advisor you
+should use the VM driver. Otherwise the docker driver is the recommented option.
+
+
+##### Using a VM driver
+
+This option uses a VM driver (like Virtualbox or kvm2) and a custom minikube
+image that contains a more recent kernel version (5.4.40) and some features
+enabled to make eBPF programs work there. More details are available
+[here](https://github.com/kinvolk/cloud-native-bpf-workshop/blob/master/minikube.md#our-branch).
+
+```
+$ wget https://cloud-native-bpf-workshop-public.s3.eu-central-1.amazonaws.com/minikube.iso
+$ minikube start --driver=kvm2 --iso-url=file://$(pwd)/minikube.iso
+
+# Deploy Inspektor Gadget in the cluster as described above
+```
+
+##### Using the docker driver
+
+This option uses docker and hence the kernel of the host.
+
+```
+$ minikube start --driver=docker
+
+# Deploy Inspektor Gadget in the cluster as described above
```

Comment:
In a future PR, we can suggest:
```
$ make minikube-install
```

So that traceloop is not started when starting with the docker driver... or maybe unify `make -C gadget-container/ minikube` and `make minikube-install` somehow?